source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2505,10 +2505,10 @@ export abstract class ContentSource extends SerializableObject {
 export class CaptionSource extends ContentSource {
     //#region Schema
 
-    static readonly languageProperty = new StringProperty(Versions.v1_1, "lang");
+    static readonly labelProperty = new StringProperty(Versions.v1_1, "label");
```

Comment:
Still needs to be mapped to v1_6, not v1_1

Reply:
Thanks for the catch! Fixed